welcome.php

```diff
@@ -48,8 +48,14 @@ class Welcome extends Module
      */
     public function __construct()
     {
+        // If the symfony container is not available this constructor will fail
+        // This can happen during the upgrade process
+        if (null == SymfonyContainer::getInstance()) {
+            return;
+        }
```

Comment:
@Quetzacoalt91 @toutantic Won't this make a broken instance? Shouldn't it be done in line 60?

Reply:
Wow, you're right.